### PR TITLE
Add Sussex-Hackers community

### PR DESCRIPTION
**Pull request by CaptainWalters**

### CODE REVIEW

File: communities/sussex-hackers/community-assessment.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+### Definition
+The current community I am a part of is hackers at Sussex University.
+Ideally this will grow to a community in and around Brighton (to connect the two universities in that area).
+  
+### Size
+The community is about 20 or so people, with low activity and low engagement.
+I would ideally like to see a larger number of students to get involved with more frequent events to foster engagement.
+The group is still less than a year old, probably 6 months and has been lacking in community leaders.
+Currently - I have no idea how to grow it.
```
Comment (joenash) [inline communities/sussex-hackers/community-assessment.md]: Reading your answers here, this isn't true! You know exactly how to grow it, you say frequently the things you'd like to see - these are things you think would make a difference. Have some confidence in yourself :)

File: communities/sussex-hackers/community-assessment.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+### Resources
+Hackathon society, external makerspace, occasional meetups, maybe a hackathon.
+I would like to see hardware resources and hardware and software workshops.
+
+### Accessibility
+Society is accessable to students of Sussex. Makerspace meets on thursdays after 8 for dropins or more availability for members.
+Meetups aren't run by students. Hackathon is organised by the society, but currently infrequent in nature.
```
Comment (joenash) [inline communities/sussex-hackers/community-assessment.md]: Could they be?

File: communities/sussex-hackers/community-assessment.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+### Interests
+The community is only CS students, but I would like to see this grow and become accessable to students of all degrees, with an interest in technology.
```
Comment (joenash) [inline communities/sussex-hackers/community-assessment.md]: How could you facilitate this?

File: communities/sussex-hackers/community-assessment.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+### Interests
+The community is only CS students, but I would like to see this grow and become accessable to students of all degrees, with an interest in technology. 
+
+### Goals
+Students hoping to get a degree and some trying to ramp-up to get a job
+- What activities and goals can you set for your community based on their existing goals?
+
+### Location
+Currently the location is online only.
```
Comment (joenash) [inline communities/sussex-hackers/community-assessment.md]: What would an offline space look like?

File: communities/sussex-hackers/community-assessment.md (hunk 2 above)
Comment (CaptainWalters) [inline communities/sussex-hackers/community-assessment.md]: Are you referring to the meetups?

**Pull request merged**